ci/pipelines/reaper/jinja.template.yml

```diff
@@ -27,6 +29,17 @@ resource_types:
     tag: latest
 
 resources:
+- name: four-times-a-day
```

Comment:
For GCE instances, we have a resource that scrapes for old TTL instances. Is there anything similar for AWS, or a way to know that 6 hrs is  the right amount of wait-time?

Reply:
The script kills after a week based on launch time, and I personally would rather not invest development time in AWS if we don't have to.